### DISCUSSION TITLE
Connection & watched as separate

### DISCUSSION THREAD

**Discussion by ghost** [trakt/showly#614] [Ideas / Feature Requests]
Update....
First I didn't notice the typo. Should be collection.

Second, showly doesn't utilize the collection.

So, that's what my suggestion is, for showly to sync collections and have a collections tab.

🤦‍♂️🤦‍♂️🤦‍♂️

When you add a movie, "add to my movies", it gets entered in trakt as watched and is put into collection. This is not correct and makes a mess out of a person's collection on trakt.

Collection is for movies the person has purchased and keeps. It is not the same as watched/history.

Please make then sperate.

**Comment by michaldrabik** [comment]
@Charles7z Hey. Showly does not use Trakt.tv Collection section at all (the green button in Trakt). There are no endpoints being used at all that reference Trakt.tv collection.

Are you certain it's coming from Showly?

**Reply by ghost** [reply]
i was just returning here to correct my comment. I got that idea in the past using showly, I think it had something to do with coming from trakt.....🤷‍♂️

So what I really want is showly to give a tab for collection 😀

I'll update the comment.

My bad!
